src/main/takeDetection/__tests__/takeDetection.test.ts

```diff
@@ -188,130 +112,22 @@ describe('findSentences should return a list of Sentence objects based on punctu
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'world!',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'Hi',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'there.',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'All',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'ok?',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'test.',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'assessment.',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-      {
-        word: 'exam',
-        duration: 1,
-        startTime: 0,
-        outputStartTime: 0,
-        originalIndex: 0,
-        pasteKey: 0,
-        bufferDurationBefore: 0,
-        bufferDurationAfter: 0,
-        deleted: false,
-        confidence: 1,
-        takeInfo: null,
-      },
-    ];
+    const wordList = makeWords([
+      'Hello',
+      'world!',
+      'Hi',
+      'there.',
+      'All',
+      'ok?',
+      'test.',
+      'assessment.',
+      'exam',
+    ]);
 
     (mockedSentenceSim as jest.Mock).mockImplementation(
       mockGetSentenceSimilarity
     );
-    const takes: InjectableTakeGroup[] = findTakes(wordList, 0.7);
+    const takes: InjectableTakeGroup[] = findTakes(wordList);
```

Comment:
Keep the hard coded 0.7 here - the reason it was put here is because relying on a constant made the test break when we changed the constant - we want to test the behaviour of the findTakes function in general, without assuming the actual threshold used in the app will be a certain value